hunting/llm/docs/aws_bedrock_ignore_previous_prompt_detection.md

```diff
@@ -9,7 +9,7 @@
 - **UUID:** `131e5887-463a-46a1-a44e-b96361bc6cbc`
 - **Integration:** [aws_bedrock.invocation](https://docs.elastic.co/integrations/aws_bedrock)
 - **Language:** `[ES|QL]`
-- **Source File:** [AWS Bedrock LLM Sensitive Content Refusals](../queries/aws_bedrock_ignore_previous_prompt_detection.toml)
+- **Source File:** [AWS Bedrock LLM Ignore Previous Prompt Detection](../queries/aws_bedrock_ignore_previous_prompt_detection.toml)
```

Comment:
@shashank-elastic - Not sure if we just did not re-generate docs or not before merging your PR, but this changed automatically without any changes to the TOML in my PR.